robot_lib.py

```diff
@@ -151,6 +152,37 @@ def dryGripper():
     daq_lib.gui_message("Dry gripper failed! " + e_s)
     setPvDesc("warmupThreshold",saveThreshold)          
     
+def DewarRefill(hours):
+  global _dewarRefillThread
+  seconds = int((hours * 60 * 60))
+  _dewarRefillThread = Thread(target=_dewarRefillTask, args=(seconds,0))
```

Comment:
could you check for the existence of the thread to prevent it being called multiple times? otherwise, you might get the function being called multiple times and having the timer being reset

Reply:
Fixed with 3e738822965449fc5567ad4a09ee9eb5b17e1f61